### ISSUE TITLE
The MSN tests should not make deprecation warnings

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8097]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8097|
|Type|enhancement|
|Created|2015-11-07 14:01:50Z|
|Branch|https://github.com/twisted/twisted/tree/msn-warnings-8097|

Right now it allows this warning to leak:

```
/buildslave/debian8-py2.7/Twisted/twisted/words/test/test_msn.py:28: DeprecationWarning: twisted.words.protocols.msn was deprecated in Twisted 15.1.0: MSN has shutdown.
  from twisted.words.protocols import msn
```

<details><summary>Searchable metadata</summary>

```
trac-id__8097 8097
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_msn_warnings_8097 branches/msn-warnings-8097
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1446904910181057 1446904910181057
changetime__1447415026174374 1447415026174374
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [46107]) Branching to msn-warnings-8097.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

This looks ok. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Ignoring all warnings is a bit brutal, for my taste, but I hope that we will remove the whole msn code soon.

+1 for the drive  by cleanup :)

Feel free to merge.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban commented|
|-|-|

Instead of ingoring all errors, can we just guard the msn usage.

Maybe I am wrong, but I think that the only place should be
```
from twisted.words.protocols import msn
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [46184]) Merge msn-warnings-8097: Stop the MSN tests raising DeprecationWarnings

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8097](https://github.com/twisted/twisted/issues/8097)
